### ISSUE TITLE
DB auto sending emails when it's not supposed to

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#356]
418904

This message went to the client and never should have because she submitted an ACH payment - which means i still have to process it and apply it to her invoice:

![image](https://cloud.githubusercontent.com/assets/8440903/7353764/21e153c4-ece4-11e4-8302-ae774bf0f24c.png)

![image](https://cloud.githubusercontent.com/assets/8440903/7353750/06092a32-ece4-11e4-902c-54a8629a480e.png)

**Comment by bradatkins** [comment]
So if a client pays by ACH they shouldn't get a confirmation email?

**Comment by ali206** [comment]
Correct. When a client pays by ACH via the DB they're not actually making the payment yet.  They're just sending me their banking information and then i have to process it and apply it to the booking.

**Comment by bradatkins** [comment]
I've made it so the message is different for ACH / CHECK. If you'd like to change the message let me know

![image](https://cloud.githubusercontent.com/assets/8440255/15696092/57d5cd16-277b-11e6-9386-98039d84b6f3.png)
